test/prides_test.dart

```diff
@@ -22,8 +22,11 @@ void main() {
   }
 
   group('Navigate the default PresentationWidget with one slide -', () {
-    final presentationWidget =
-        PresentationWidget(slides: const [MockSlideWidget()]);
+    final presentationWidget = PresentationWidget(
+      slides: const [
+        MockSlideWidget(),
```

Comment:
You don't need to add trailing commas everywhere in the file. It will pass even without this change. You only need to change the part of code you added. :)

Trailing commas are used only when the line exceeds 80 characters. If it doesn't, we don't need to.

Thank you.